app.js

```diff
@@ -4,7 +4,7 @@ const app = express();
 const {manage404Errors,manage400Errors,manageCustomErrors} = require('./errorhandling')
 const {getMessage, getTopics} = require ('./controllers/topics.controller');
 const {getArticles, getArticleById,patchVotesInArticles} = require ('./controllers/article.controller');
-const {getCommentsByArticleId,postComment} = require ('./controllers/comments.controller');
+const {getCommentsByArticleId,postComment,deleteComment} = require ('./controllers/comments.controller');
```

Comment:
Really like the consistent naming pattern you've chosen for the controllers and how you've split up the article and comments controllers to seperate files to make the code easier to maintain!